test/parallel/test-net-server-close-before-dns.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
+const net = require('net');
+
+net.createServer(() => {}).listen(8080, 'localhost', () => {
+  net.connect(8080, () => {
+    throw new Error('a connection was made');
+  }).on('error', (e) => process._rawDebug('not able to connect:', e.message));
```

Comment:
this shouldn't have reason to write out to console, especially using `process._rawDebug`